src/Engine.php

```diff
@@ -130,8 +130,8 @@ private function runInitialTestSuite(): void
 
         $initialTestSuitProcess = $this->initialTestsRunner->run(
             $this->config->getTestFrameworkExtraOptions(),
-            $this->config->shouldSkipCoverage(),
-            explode(' ', (string) $this->config->getInitialTestsPhpOptions())
+            explode(' ', (string) $this->config->getInitialTestsPhpOptions()),
```

Comment:
I'm not sure this change is needed (swapping parameters)

Reply:
no but I also dislike having the parameters declared in a certain order in the method and another order for the properties declaration, assignment and method orders, so just made sure they are all in the same order

Reply:
Actually that's not the case here so nvm, will revert that

Reply:
It was to keep the same order as `TestFrameworkAdapter::getInitialTestRunCommandLine`